src/helpers/Monitor.hpp

```diff
@@ -196,11 +198,16 @@ class CMonitor {
         return vecPosition == rhs.vecPosition && vecSize == rhs.vecSize && szName == rhs.szName;
     }
 
+    // workspace previous per monitor functionality
+    SWorkspaceIDName getPrevWorkspaceIDName(const WORKSPACEID id);
+    void             addPrevWorkspaceID(const WORKSPACEID id);
+
   private:
-    void        setupDefaultWS(const SMonitorRule&);
-    WORKSPACEID findAvailableDefaultWS();
+    void                    setupDefaultWS(const SMonitorRule&);
+    WORKSPACEID             findAvailableDefaultWS();
 
-    bool        doneScheduled = false;
+    bool                    doneScheduled = false;
+    std::stack<WORKSPACEID> m_prevWorkSpaces;
```

Comment:
what is this capitalization lol, `prevWorkspaces`

Reply:
done